sdk/include/reactos/shellutils.h

```diff
@@ -378,6 +378,28 @@ HRESULT inline ShellObjectCreatorInit(T1 initArg1, T2 initArg2, T3 initArg3, T4
     return hResult;
 }
 
+template<class T, class T1, class T2, class T3, class T4, class T5>
```

Comment:
Time for variadic templates!

(just kidding, our compilers won't support those)